cloud_edition/flexibility_mode/docs/partner_starterkit.rst

```diff
@@ -0,0 +1,43 @@
+Partners Starterkit
+===================
+
+After the Flexibility setup, thanks you for taking notice of these followings points.
```

Comment:
After the Flexibility setup, you will need to take notice of the following points: